cmd/zrepl/zrepl.c

```diff
@@ -173,17 +174,23 @@ uzfs_zvol_io_receiver(void *arg)
 	/* First command should be OPEN */
 	while (zinfo == NULL) {
 		if (open_zvol(fd, &zinfo) != 0) {
+			if ((zinfo != NULL) &&
+			    (zinfo->is_io_ack_sender_created))
+				goto exit;
```

Comment:
why this piece of code was added? zinfo cannot be != NULL if return code is != 0.

Reply:
you are right 👍